index.html

```diff
@@ -4,14 +4,28 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <title>Aaron Keegan Developer Portfolio</title>
 </head>
 <body>
-
-  <!-- header
-  name:Aaron, sandwich icon -->
-  <!-- Section h1, h3, p (about) icons,
-  background image -->
-  
+  <header>
+    <nav class="top-nav">
+      <div class="logo">Aaron Keegan</div>
```

Comment:
Please wrap your name with an `a` tag because this will act as a link in the future to return to home 😎